src/sentry/options/defaults.py

```diff
@@ -2718,3 +2718,18 @@
     default=10000,
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
+register(
+    "celery_split_queue_legacy_mode",
+    default=["post_process_transactions"],
+    flags=FLAG_AUTOMATOR_MODIFIABLE,
+)
+register(
+    "celery_split_queue_rollout",
+    default={"post_process_transactions": 1.0},
```

Comment:
Tasks will still be routed to the default queue after this PR has been deployed, because `celery_split_queue_task_rollout` is empty, right?

Reply:
So you would think, and you will be right. 
sadly (or fortunately as that was in response to an incident) there is an override in the ops repo that added that setting, and it is set in both US and DE.
https://github.com/getsentry/ops/blob/master/k8s/services/getsentry/templates/k8s_regionsilo_ops.py.j2#L20-L26

Tasks on post process forwarder have already been routed to the split queues for two weeks.